### ISSUE TITLE
scatterplot.render bug

### ISSUE THREAD

**Issue by xyx650** [tensorflow/tfjs#3415]
#### chrome ver 83.0.4103.97

i'm running the demo code in the chrome, the console print 'Cannot read property 'length' of undefined '

the demo code at 'https://js.tensorflow.org/api_vis/1.4.0/#render.scatterplot' is:

```javascript
import * as tfvis from '@tensorflow/tfjs-vis'; // this is my import

const series1 = Array(100).fill(0)
   .map(y => Math.random() * 100 - (Math.random() * 50))
   .map((y, x) => ({ x, y, }));

const series2 = Array(100).fill(0)
   .map(y => Math.random() * 100 - (Math.random() * 150))
   .map((y, x) => ({ x, y, }));

const series = ['First', 'Second'];
const data = { values: [series1, series2], series }

const surface = { name: 'Scatterplot', tab: 'Charts' };
tfvis.render.scatterplot(surface, data);
```

here is the error message print on console:

Uncaught (in promise) TypeError: Cannot read property 'length' of undefined
    at expression (expression.js:6)
    at Object.operator (expression.js:25)
    at Context.operatorExpression (context.js:147)
    at Context.parseOperator (operator.js:12)
    at dataflow.js:24
    at Array.forEach (<anonymous>)
    at Context._default [as parse] (dataflow.js:24)
    at _default (runtime.js:6)
    at new View (View.js:76)
    at Object.<anonymous> (embed.ts:276)

i think isn't fjs-vis version not match it's dependencies version ?

sorry for my broken english...

**Comment by rthadur** [comment]
Changes have been merged and new changes will be available in next release.Thanks for reporting.

**Comment by tafsiri** [comment]
@xyx650 I just released tfjs-vis 1.4.1 to NPM give that a try and let us know if it works.

**Comment by tafsiri** [comment]
Looks like that didn't fix it. Lets continur to track this in this issue https://github.com/tensorflow/tfjs/issues/3397

**Comment by xyx650** [comment]
I updated to 1.4.2, and it didn't work~
the same demo code, and the error message is:

error.js:2 Uncaught (in promise) Error: Unrecognized transform type: "filter"
    at _default (error.js:2)
    at _default (transform.js:12)
    at data.js:12
    at Array.forEach (<anonymous>)
    at parseData (data.js:11)
    at scope.js:26
    at Array.forEach (<anonymous>)
    at _default (scope.js:26)
    at parseView (view.js:65)
    at Object._default (parse.js:12)

I will have a try for another version~~
Thanks for answering

**Comment by xyx650** [comment]
@tafsiri I had no found any about vega message, I just run the simple demo code with parcel on chrome.

I had try the Ver.1.41, the error message as same as 1.42

**Comment by tafsiri** [comment]
could you clear your lock files and node_modules and try again?

**Comment by xyx650** [comment]
I had cleared both, whether I use npm or yarn to install, the error still appears

**Comment by xyx650** [comment]
@tafsiri Hi, did you run without any errors @1.4.1/1.4.2?

**Comment by tafsiri** [comment]
@xyx650 Hi. Forgot to update this. We released 1.4.3 to help address this. Give that a try, if that doesn't work, could you make a runnable codepen example reproducing the bug?

**Comment by xyx650** [comment]
@tafsiri Oh, it works at 1.4.3, thanks a lot
